RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -254,9 +254,12 @@
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
 for e in [pp_on_XeXe_2017, pp_on_AA_2018]:
     e.toModify(photonConvTrajSeedFromSingleLeg,
-               RegionFactoryPSet = dict(RegionPSet = dict(ptMin = 999999.0,
-                                                          originRadius = 0,
-                                                          originHalfLength = 0)
+               RegionFactoryPSet = dict(RegionPSet = dict(originRadius = 0,
+                                                          originRScaling4BigEvts = cms.bool(True),
+                                                          minOriginR = cms.double(0),
+                                                          scalingStartNPix = cms.double(0),
+                                                          scalingEndNPix = cms.double(1)#essentially turn off immediately 
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
If I remove the type specifiers, the code then does not run.  For example, changing the first line to 'originRScaling4BigEvts = True,' causes the following error:

'Unknown parameter name originRScaling4BigEvts specified while calling Modifier'

Do you happen to know the reason for this?  From quickly checking the parameter names in [1], I do not think I have a naming mistake.

[1] https://github.com/cms-sw/cmssw/blob/master/RecoTracker/TkTrackingRegions/plugins/GlobalTrackingRegionWithVerticesProducer.h#L45

Reply:
this looks suspicious.
as if the photonConvTrajSeedFromSingleLeg.RegionFactoryPSet is not coming from the right place or/that it would even know what originRScaling4BigEvts parameter is.

Reply:
The `RegionFactoryPSet` is set here
https://github.com/cms-sw/cmssw/blob/950fd72c5d55ef0cb2a452647151a94ee4f0f246/RecoTracker/ConversionSeedGenerators/python/PhotonConversionTrajectorySeedProducerFromSingleLeg_cfi.py#L29-L37
(and I don't see any later overrides for these modifiers)
but the region producer is `GlobalTrackingRegionProducerFromBeamSpot` and not ` GlobalTrackingRegionWithVerticesProducer`.

@abaty, if you want the latter here, you should customize it here
https://github.com/abaty/cmssw/blob/ee8079cec93143973f0f3c15adfc542094233c96/RecoTracker/ConversionSeedGenerators/python/PhotonConversionTrajectorySeedProducerFromSingleLeg_cfi.py#L72-L78